presto-main/src/main/java/com/facebook/presto/server/CoordinatorModule.java

```diff
@@ -175,6 +175,7 @@ protected void setup(Binder binder)
         jsonCodecBinder(binder).bindJsonCodec(TaskInfo.class);
         jsonCodecBinder(binder).bindJsonCodec(QueryResults.class);
         jaxrsBinder(binder).bind(StatementResource.class);
+        newExporter(binder).export(StatementResource.class).withGeneratedName();
```

Comment:
The commit title does not conform to our guidelines (the PR title does).
Suggestion: `Add counter for POST requests to statement resource`

Body suggestion:
```
This counter makes it easier to monitor large spikes in the incoming query rate.
This counter is incremented prior to potentially expensive operations in 
SqlQueryManager (which has its own counters).
```